src/System.Runtime.WindowsRuntime/src/System/Resources/WindowsRuntimeResourceManager.cs

```diff
@@ -231,7 +231,20 @@ private static void GlobalResourceContextChanged(object sender, IMapChangedEvent
             Debug.Assert(s_globalResourceContextFallBackList != null);
             Debug.Assert(s_globalResourceContext != null);
 
-            List<String> languages = new List<string>(s_globalResourceContext.Languages);
+            IReadOnlyList<string> langs;
+
+            try
+            {
+                langs = s_globalResourceContext.Languages;
+            }
+            catch (ArgumentException)
+            {
+                // Sometimes Windows Runtime fails and we get Argument Exception which can fail fast the whole app
+                // to avoid that we ignore the exception.
```

Comment:
Thanks.  This seems reasonable to unblock us for now, but it'd be good to better understand this, and either put a real fix in place on the .NET side to prevent such corruption in the first place, or ensure Windows is tracking a fix if it's actually on the WinRT side.  In the meantime, this should have a TODO with an issue number to follow-up.